compiler/src/dotty/tools/dotc/core/StdNames.scala

```diff
@@ -459,6 +459,7 @@ object StdNames {
     val materializeWeakTypeTag: N = "materializeWeakTypeTag"
     val materializeTypeTag: N   = "materializeTypeTag"
     val mirror : N              = "mirror"
+    val meta : N                = "meta"
```

Comment:
We need to change to `macro`? No, we can just remove this line, as `macro` already exists.

Reply:
Done.